src/components/notifications/Discord.vue

```diff
@@ -16,4 +16,46 @@
         <label for="discord-prefix-message" class="form-label">{{ $t("Prefix Custom Message") }}</label>
         <input id="discord-prefix-message" v-model="$parent.notification.discordPrefixMessage" type="text" class="form-control" autocomplete="false" :placeholder="$t('Hello @everyone is...')">
     </div>
+
+    <div class="mb-3">
+        <label for="discord-message-type" class="form-label">{{ $t("Select message type") }}</label>
+        <br>
+        <select id="discord-message-type" v-model="$parent.notification.discordChannelType" class="form-select">
+            <option value="channel">{{ $t("Send to channel") }}</option>
+            <option value="createNewForumPost">{{ $t("Create new forum post") }}</option>
+            <option value="postToThread">{{ $t("Post to existing thread / forum post") }}</option>
+        </select>
+    </div>
+
+    <div v-if="$parent.notification.discordChannelType === 'createNewForumPost'">
+        <div class="mb-3">
+            <label for="discord-target" class="form-label">{{ $t("Forum post name") }}</label>
+            <input id="discord-target" v-model="$parent.notification.postName" type="text" class="form-control" autocomplete="false">
+            <div class="form-text">
+                {{ $t("whatHappensAtForumPost") }}"{{ $t("Post to existing thread / forum post") }}"
+            </div>
+        </div>
+    </div>
+    <div v-if="$parent.notification.discordChannelType === 'postToThread'">
+        <div class="mb-3">
+            <label for="discord-target" class="form-label">{{ $t("Thread / Forum post ID") }}</label>
+            <input id="discord-target" v-model="$parent.notification.threadId" type="text" class="form-control" autocomplete="false" :placeholder="$t('e.g. {discordThreadID}', { discordThreadID: 1177566663751782411 })">
+            <div class="form-text">
+                {{ $t("wayToGetDiscordThreadId") }}
```

Comment:
Currently, this is not quite translatable, as some languages might not have this word in this location.
Could you use [`<i18n-t ...`](https://vue-i18n.intlify.dev/guide/advanced/component.html) instead?

Reply:
You mean like this?:
```
<i18n-t tag="div" keypath="wayToGetDiscordThreadId" class="form-text">
    <a
         href="https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-"
         target="_blank"
     >{{ $t("here") }}</a>
</i18n-t>
```

Reply:
@CommanderStorm I've made the adjustments, hopefully as intended by you.